src/hooks/useENSRegistrationForm.ts

```diff
@@ -148,6 +148,13 @@ export default function useENSRegistrationForm({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [isEmpty(defaultRecords), updateRecords]);
 
+  // Reset errors if changedRecords is reset
+  useEffect(() => {
+    if (isEmpty(changedRecords)) {
+      setErrors({});
```

Comment:
wondering if this should have been working here https://github.com/rainbow-me/rainbow/pull/3904/files#diff-414802e989ced2afc915f2f93852ceec4c3b28faaab72b6a9f6168dfd0f903beR225 while changing inputs instead of forcing a reset?

Reply:
![image](https://user-images.githubusercontent.com/12115171/182697623-fcccc94d-5cc1-470d-a7f8-c48b772df52c.png)
picture of the diff

Reply:
+1

Reply:
i don't think you are explicitly changing the field via the event handler though... ben's solution may be correct

Reply:
yes, `onChangeField` does not get called with `changedRecords` state is reset